site/_posts/2017-03-28-pontificatory-requestiveous.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Last Week in ManageIQ: Pontificatory Requestiveous - PR"
+author: syncrou
+date: 2017-03-28 09:00:00 -04:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi, [Dr. Drew Syncrounious](https://github.com/syncrou) here to talk with you about an important new solution just released into 
+[master](https://github.com/manageiq/manageiq). Known scientifically as Pontificatory Requestiveous, for us in the industry, it is affectionately called:
+
+[PR](https://help.github.com/articles/about-pull-requests/).
+
+A few questions
+
+Do you struggle with [memory loss](http://stackoverflow.com/questions/23281331/how-to-clean-up-allocated-memory-in-c)?
+
+Overly terse [varible names](https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js)?
```

Comment:
@syncrou did you need to modify something based on Drew's comment before we merge?